dist/backbone.intercept.js

```diff
@@ -78,9 +78,9 @@
         return;
       }
   
-      // If the Ctrl key is held, we want to open in a new tab.
-      bypass = e.ctrlKey;
-      if (bypass) {
+      // If the platform’s modifier key is held, we want to open in a new tab.
+      var platform = navigator.platform.toLowerCase();
+      if((~platform.indexOf('win') && e.ctrlKey) || (~platform.indexOf('mac') && e.metaKey)) {
```

Comment:
Not a big fan of using `~`. Its succinct, which is nice, but not very expressive. I think code is read more than written, so I opt for expressiveness over succinctness as a rule o' thumb.